src/clients/BundleDataClient.ts

```diff
@@ -912,7 +913,12 @@ export class BundleDataClient {
         // by checkings its on-chain fill status.
         const fillStatus: BigNumber = await spokePoolClients[deposit.destinationChainId].spokePool.fillStatuses(
```

Comment:
`sdkUtils.relayFillStatus()` will handle the bounds checking and return a `FillStatus` enum - might be nice to use that (will probably also collapse the code to a single line).

```suggestion
        const fillStatus = await relayFillStatus(
```